core/src/main/scala/scalaz/Void.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz
+
+import scalaz.zio.{Void => ZioVoid}
+
+trait VoidModule {
+  type Void
+
```

Comment:
I think maybe want we want to do here is expose the type.

So maybe: `type Void = ZioVoid`. Then delete `VoidImpl`.

The reason is that then Scalaz will have the "same" Void as ZIO, which will result in better interop.

Reply:
Okay then implementation will be provided by `VoidModule` itself. right?

so in package object we will have something like..
```
val Void: VoidModule = new VoidModule {}
type Void = Void.Void
```

Reply:
Yes, I'd make it an object.